### ISSUE TITLE
check-line-alignment breaks if there is no @param type

### ISSUE THREAD

**Issue by MiK546** [gajus/eslint-plugin-jsdoc#714]
The `check-line-alignment` rule, when run with the `never` option, seems to give an error about lines being aligned even if there is only one space between the `@param` tag, the parameter name and its description. If I add the type definition curly braces between `@param` and the parameter name the error disappears.

## Expected behavior
No errors are given.

## Actual behavior
The following error is given with the test code below:

```
/file/path/file.js
  2:0  error  Expected JSDoc block lines to not be aligned  jsdoc/check-line-alignment

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

## ESLint Config
```json
{
    "plugins": [
        "jsdoc"
    ],
    "rules": {
        "jsdoc/check-line-alignment": [
            2,
            "never"
        ]
    }
}
```

## ESLint sample
```js
/**
 * @param parameter Description.
 */
function func(parameter){

}
```

## Environment

- Node version: 12.22.1
- ESLint version 7.25.0
- `eslint-plugin-jsdoc` version: 32.3.1
